haskell/test/SMT.hs

```diff
@@ -406,8 +409,8 @@ generateTokenOwnerAction idx = do
       operator <- getRandomOperator
       owner <- getRandomOwner
       updateOperation <- lift $ elements
-        [ FA2.Add_operator (#owner .! owner, #operator .! operator)
-        , FA2.Remove_operator (#owner .! owner, #operator .! operator)
+        [ FA2.Add_operator (#owner .! owner, (#operator .! operator, #token_id .! 0))
+        , FA2.Remove_operator (#owner .! owner, (#operator .! operator, #token_id .! 0))
```

Comment:
Why not generate an action with invalid token_id and check that it's rejected?

Reply:
Included.